site/pages/docs/ref/wb-jsonmanager/wb-jsonmanager-en.hbs

```diff
@@ -232,402 +243,405 @@
-<td><code>{ "extractor": [ { "selector": "h2" } ] &hellip;}</code> or <code>{ "extractor": [ { "selector": "dl[data-dl]" } ] &hellip;}</code></td>
-<td>A valid css selector for any html element from the page </td>
-</tr>
-<tr>
-<td><code>path</code></td>
-<td><code>{ "extractor": [ { "path": "firstH2" } ] &hellip;}</code> or <code>{ "extractor": [ { "path": "myGroup" } ] &hellip;}</code></td>
-<td>The value of this property will be the pointer in the JSON object dataset that will be created</td>
-</tr>
-<tr>
-<td><code>selectAll</code></td>
-<td><code>{ "extractor": [ { "selectAll": true } ] &hellip;}</code> </td>
-<td>If true, it will search in the page for  all the html elements indicated in the <code>selector</code> property</td>
-</tr>
-<tr>
-<td><code>attr</code></td>
-<td><code>{ "extractor": [ { "attr":"data-td" } ] &hellip;}</code> </td>
-<td>If present, it will collect the value from the specified attribute for the mentioned selector instead of the text value of the <code>selector</code> property</td>
-</tr>
-<tr>
-<td><code>interface</code></td>
-<td><code>{ "extractor": [ { "interface":"referer" } ] &hellip;}</code> or <code>{ "extractor": [ { "interface":"locationHref" } ] &hellip;}</code> </td>
-<td>If present, it will collect the value from predefined properties as follows <span class="text-muted">(This list is not exhaustive):</span>
-	<ul><li><code>referer</code> is assigned for the <code>document.referrer</code> value</li><li><code>locationHref</code> is assigned for <code>location.href</code> value</li></ul></td>
-</tr>
-<tr>
-<td><code>extractor</code></td>
-<td><code>{ "extractor": [ { "extractor":[ {
-									"selector": "dt",
-									"path": "title",
-									"attr":"data-td"
-								},
-								{
-									"selector": "dd",
-									"path": "desc"
-								}] } ] &hellip;}</code> </td>
-<td>Used for combined selectors e.g. description lists. Same configuration as for outer <code>extractor</code> option, it is an array of objects containing html selectors with their specific paths.
-
-</tr>
-</tbody>
-</table>
+	<h2>Configuration options</h2>
+	<table class="table">
+		<thead>
+			<tr>
+				<th>Option</th>
+				<th>Description</th>
+				<th>How to configure</th>
+				<th>Values</th>
+			</tr>
+		</thead>
+		<tbody>
+			<tr>
+				<td><code>url</code></td>
+				<td><strong>Required</strong>. Define the url or the dataset name to use. When used in a template mode, the URL should point to an array object.</td>
+				<td>You can follow the url or the dataset name by a <a href="https://tools.ietf.org/html/rfc6901">JSON Pointer (RFC6901)</a>.</td>
+				<td>
+					<dl>
+						<dt><code>data-wb-jsonmanager='{ "url": "location/of/json/file.json#/" }'</code></dt>
+						<dd>The url are a json file</dd>
+						<dt><code>data-wb-jsonmanager='{ "url": "[datasetName]#/" }'</code></dt>
+						<dd>The url is a reference to a dataset managed by a json-manager defined in the same page</dd>
+					</dl>
+				</td>
+			</tr>
```

Comment:
Complement the description to say that we can nom fetch multiple JSON file + an additional code sample in the description list.